app/next-i18next.config.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @file You shouldn't have to worry about this file. Mainly here so imports of the config file
+ * (in tests or Storybook) have the correct type for the config object.
+ */
+declare const _exports: import("next-i18next").UserConfig;
+export = _exports;
```

Comment:
Good question. I didn't actually look into this very deeply. This file was generated by running the following script, which was what was in the [TypeScript docs here](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html):

```sh
npx -p typescript tsc app/next-i18next.config.js --declaration --allowJs --emitDeclarationOnly --outDir app/types
```

Looking into this a bit more, I found [this](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html):

> Note that using `export default` in your .d.ts files requires [esModuleInterop: true](https://www.typescriptlang.org/tsconfig#esModuleInterop) to work. If you can’t have `esModuleInterop: true` in your project, such as when you’re submitting a PR to Definitely Typed, you’ll have to use the `export=` syntax instead. This older syntax is harder to use but works everywhere.

Since our project _does_ have `esModuleInterop: true`, it looks like I can manually update the file to use `export default` instead: https://github.com/navapbc/template-application-nextjs/pull/211/commits/657fa37c5c6918c735ddab00ac25a25fecc11145

Reply:
Apologies for not quite understanding. What's the purpose of [these lines](https://github.com/navapbc/template-application-nextjs/blob/main/app/next-i18next.config.js#L45-L47)? They seem redundant if we have this .d.ts file. Or do we need both those lines and this file?